data/weapons/weapon_data.csv

```diff
@@ -1,59 +1,59 @@
+Micro Missile Cluster,diableavionics_microcluster_left,3,,7500,3000,,50,,5,0,,90,2.5,10,HIGH_EXPLOSIVE,,,0.2,3.8,15,0.1,6,6,5,6,,250,150,18,50,,,"SYSTEM, DO_NOT_AIM, DO_NOT_CONSERVE, DO_NOT_AIM",,,,,,,,,,,,,,,
+Micro Missile Cluster,diableavionics_microcluster2,3,,7500,3000,,50,,5,0,,90,2.5,10,HIGH_EXPLOSIVE,,,0.2,3.8,15,0.1,6,6,5,6,,250,5,21,50,,,"SYSTEM, DO_NOT_AIM, DO_NOT_CONSERVE, DO_NOT_AIM",,,,,,,,,,,,,,,
+Micro Missile Array,diableavionics_microarray,3,,17500,3000,,50,,5,0,,960,8,32,HIGH_EXPLOSIVE,,,0.1,3.9,80,0.1,6,6,5,6,,250,150,18,50,,,"SYSTEM, DO_NOT_AIM, DO_NOT_CONSERVE, DO_NOT_AIM",,,,,,,,,,,,,,,
+Micro Missile Array,diableavionics_microarrayM,3,,17500,3000,,50,,5,0,,480,4,16,HIGH_EXPLOSIVE,,,0.1,3.9,40,0.1,6,6,5,6,,250,150,18,50,,,"SYSTEM, DO_NOT_AIM, DO_NOT_CONSERVE, DO_NOT_AIM",,,,,,,,,,,,,,,
 ,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
-Magicbox PD Missile,diableavionics_magicbox,1,0.75,400,1000,,150,,3,15,5,8,0.15,1,FRAGMENTATION,25,,0.5,1,1,0,7,14,14,12,,475,35,3.7,40,,,"DO_NOT_AIM, DO_NOT_CONSERVE, PD_ONLY, PD","pd9, missile0, diable_weapon_bp, rare_bp",,Diable Avionics,,Point Defense (Area),Excelent,Excelent,,,,,,,,11.23
+Magicbox PD Missile,diableavionics_magicbox,1,0.75,400,1200,,150,,3,15,5,12,0.5,3,FRAGMENTATION,25,,0.5,1,1,0,7,14,14,12,,475,35,3.7,40,,,"DO_NOT_AIM, DO_NOT_CONSERVE, PD_ONLY, ","pd9, missile0, diable_weapon_bp, rare_bp",,Diable Avionics,,Point Defense (Area),Excelent,Excelent,,,,,,,,11.23
 ,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
-Thunderbolt MRM Rack,diableavionics_thunderboltsmall,0,,200,1800,,800,,5,6,6,8,,,HIGH_EXPLOSIVE,0,,0,5.5,2,0.5,6,9,9,9,,300,50,10,120,,,DO_NOT_AIM,"missile7, strike4, diable_weapon_bp",,Diable Avionics,,Finisher,Medium,Good,,,"%s: spreads up to %s of its damage inside the enemy ship, disabling weapons away from the edge of the hull.",Pierce Damage|half,,,TRUE,11.4
-Thunderbolt MRM Pod,diableavionics_thunderboltpod,1,,1150,2000,,800,,5,6,10,16,,,HIGH_EXPLOSIVE,0,,0.4,6,4,0.4,0,12,4,12,,300,650,10,120,,,DO_NOT_AIM,"missile13, strike9, diable_weapon_bp, rare_bp",,Diable Avionics,,Finisher,Medium,Good,,,"%s: spreads up to %s of its damage inside the enemy ship, disabling weapons away from the edge of the hull.",Pierce Damage|half,,,TRUE,
+Thunderbolt MRM Rack,diableavionics_thunderboltsmall,0,,200,1800,,800,,5,6,6,8,,,HIGH_EXPLOSIVE,0,,0,5.5,2,0.5,6,9,9,9,,400,50,10,250,,,DO_NOT_AIM,"missile7, strike4, diable_weapon_bp",,Diable Avionics,,Finisher,Medium,Good,,,"%s: spreads up to %s of its damage inside the enemy ship, disabling weapons away from the edge of the hull.",Pierce Damage|half,,,TRUE,11.4
+Thunderbolt MRM Pod,diableavionics_thunderboltpod,1,,1150,2500,,800,,5,6,14,24,,,HIGH_EXPLOSIVE,0,,0.4,6,4,0.4,0,12,4,12,,400,650,10,250,,,DO_NOT_AIM,"missile13, strike9, diable_weapon_bp, rare_bp",,Diable Avionics,,Finisher,Medium,Good,,,"%s: spreads up to %s of its damage inside the enemy ship, disabling weapons away from the edge of the hull.",Pierce Damage|half,,,TRUE,
```

Comment:
Thunderbolt MRM Rack (small) projectile speed max set to 400 (from 300), projectile health set to 250 (from 120).
Thunderbolt MRM Pod (medium) range increased to 2500 (from 2000), OP cost increased to 14 (from 10), ammo increased to 24 (from 16), projectile speed max set to 400 (from 300), projectile health set to 250 (from 120).